routes/recognizer_routes.py

```diff
@@ -41,4 +44,40 @@ def recognize_object_from_zip():
     except Exception as e:
         return jsonify({"success": False, "message": "An error occurred", "error": str(e)}), 500
 
-#TODO nuevo endpoint devolver el json de classes_yolo (GET)
\ No newline at end of file
+@face_recognition_blueprint.route('/face_recognition', methods=['POST'])
+def recognize_from_zip():
+    print("---INICIANDO FACE RECOGNITION---", flush=True)
+    data = request.form
+    zip_url = data.get('zip_url')
+    zip_filename =  download_file_from_url(zip_url)
+    print(zip_filename)
+    model_type = data.get('model_type')
+    confidence_threshold = (data.get('confidence_threshold', 0.1))
+    word = data.get('word')
+    image_file_reference = request.files.get('image_file_reference')
+    print("guardara la imagen")
+    print(image_file_reference)
+    reference_path = save_image(image_file_reference)
+    print("guardo la imagen")
+
+    try:
```

Comment:
TODO: Try Except code could be replace with a method placed in utils to reduce the logic in the endpoint